src/internet_identity/src/lib.rs

```diff
@@ -1,5 +1,4 @@
 //! Various APIs for managing internet identities.
```

Comment:
This comment is no longer accurate, maybe remove it?

It would be nice if we could get rid of the whole `lib.rs` file, but I think the `get_random` fix below only works if placed in the `lib.rs` file for some reason...

Reply:
Removed the comment. 
But indeed, it seems we still need `lib.rs`.